LayoutTests/accessibility/mac/html-section-elements.html

```diff
@@ -44,8 +49,10 @@
           shouldBe("body.childAtIndex(6).childAtIndex(0).subrole", "'AXSubrole: '");
 
           // When a header and footer are inside a section, they should not be landmarks.
-          // Instead, expect only the static text inside to be exposed.
-          shouldBe("body.childAtIndex(7).role", "'AXRole: AXStaticText'");
```

Comment:
We probably still want to verify the role of `childAtIndex(7)` (which is the `section > header`. Seems like it becomes a group after this change, which is fine since `section > footer` is a group.

Reply:
I think body.ChildAtIndex(7) is pointnig the <section>. After this change, it becomes AXGroup.

Reply:
I ran this test with your change and added DOM `id`s, and I'm seeing `childAtIndex(7)` as `section > header`.